CHANGES.rst

```diff
@@ -20,14 +44,48 @@ Bug Fixes
 
 - Fixes to the jwst loaders. [#759]
 
+- Fix handling of ``SpectralCollection`` objects in moment calculations. [#781]
+
+- Fix issue with non-loadable x1d files. [#775]
+
+- Fix WCS handling in SDSS loaders. [#738]
+
+- Fix the property setters for radial velocity and redshift. [#722]
+
+- Fix line test errors and include python 3.9 in tests. [#751]
+
+- Fix smoothing functionality dropping spectrum meta information. [#732]
+
+- Fix region extraction for ``Spectrum1D`` objects with multi-dimensional fluxes. [#724]
+
 Documentation
 ^^^^^^^^^^^^^
 
+- Update SDSS spectrum documentation examples. [#778]
+
+- Include new documentation on working with ``SpectralCube`` objects. [#726, #784]
+
+- Add documentation on spectral cube related functionality. [#783]
+
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 - Improved error messages when creating ``SpectralRegion`` objects. [#759]
 
+- Improve docstrings and intialization of ``SpectralRegion`` objects. [#770]
+
+- Prevent CI from fast-failing on test failure. [#752]
+
+- Update documentation favicons and ensure color consistency. [#780]
+
+- Remove fallback ``SpectralCoord`` code and rely on upstream. [#786]
+
+- Move remaining loaders to use utility functions for parsing files. [#718]
+
+- Remove unnecessary data reshaping in tabular fits writer. [#730]
+
+- Remove astropy helpers and CI helpers dependencies. [#562]
+
```

Comment:
(goes with my suggestion  above, although this one could be skipped to just leave it out of the changelog.)
```suggestion

- Move from Travis to GitHub actions. [#736]
```